README.md

```diff
@@ -36,19 +36,19 @@ The specific ONNX release version that we support in the master branch of ONNX-T
 
 To install the latest version of ONNX-TF via pip, run `pip install onnx-tf`.
 
-Because users often have their own preferences for which variant of Tensorflow to install (i.e., a GPU version instead of a CPU version), we do not explicitly require tensorflow in the installation script. It is therefore users' responsibility to ensure that the proper variant of Tensorflow is available to ONNX-TF. Moreoever, we require Tensorflow version >= 1.15.0.
+Because users often have their own preferences for which variant of Tensorflow to install (i.e., a GPU version instead of a CPU version), we do not explicitly require tensorflow in the installation script. It is therefore users' responsibility to ensure that the proper variant of Tensorflow is available to ONNX-TF. Moreoever, we require Tensorflow version == 1.15.0. For Tensorflow 2.0 support please refer [here](https://github.com/onnx/onnx-tensorflow/blob/master/README.md/).
```

Comment:
Since Tensorflow 2.0 is not supported in any onnx-tf official releases, we probably should remove the last sentence for now. The master branch is currently under development for TF 2.0 so the reference could be in the next section.

Reply:
Done